templates/variant-track-desc.csv

```diff
@@ -0,0 +1,7 @@
+Genome_UUID,Track_name,Source_name,Source_URL,Description
+a733574a-93e7-11ec-a39d-005056b38ce3,SGRP short variants,SGRP,http://www.moseslab.csb.utoronto.ca/sgrp,"Variation features from SGRP, with Ensembl identifiers"
```

Comment:
```suggestion
a733574a-93e7-11ec-a39d-005056b38ce3,EVA short variants,EVA,https://www.ebi.ac.uk/eva,"All short variants (SNPs and indels) data from European Variation Archive (EVA) - release 6"
```

Or probably we do not need this here as it is standard EVA source.

Reply:
Good spot. Yes, we can just remove this CSV row as it doesn't differ from the eva track template.